shelley/chain-and-ledger/shelley-spec-ledger-test/test/Test/Shelley/Spec/Ledger/Serialisation/Golden/Address.hs

```diff
@@ -48,6 +49,8 @@ import Test.Tasty (TestTree)
 import qualified Test.Tasty as T
 import qualified Test.Tasty.HUnit as T
 
+import Cardano.Prelude (decodeEitherBase16)
```

Comment:
I thought we discussed we were going to use the utilities from `base16-bytestring` directly? I'd rather avoid adding more dependencies on `Cardano.Prelude`

Reply:
The new `decodeEitherBase16` behaves exactly like `decode` from `base16-bytestring-1.0.0.0`, which will make dropping `cardano-prelude` easy to do.

The intent is the minimise risk.  I wanted to be sure downstream projects don't use the `decode` function in a surprising way that relies on the old behaviour.

When I'm certain there are no surprises in downstream projects, I'll drop the upper bound and all the projects will be free switch to the `decode` in the new version of `base16-bytestring` and drop `cardano-prelude`.